nibabel/arrayproxy.py

```diff
@@ -116,6 +115,10 @@ def __init__(self, file_like, spec, *, mmap=True, keep_file_open=None):
             True gives the same behavior as ``mmap='c'``.  If `file_like`
             cannot be memory-mapped, ignore `mmap` value and read array from
             file.
+        order : {'F', 'C'}, optional, keyword only
```

Comment:
Shouldn't this be:

```suggestion
        order : {None, 'F', 'C'}, optional, keyword only
```

?